app/v1/connectors/ListChargesConnector.scala

```diff
@@ -19,7 +19,7 @@ package v1.connectors
 import config.AppConfig
 import javax.inject.{Inject, Singleton}
 import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
```

Comment:
Can you please revert back to use previous import as this import is deprecated. Info can be seen [here](https://github.com/hmrc/bootstrap-play/blob/master/bootstrap-backend-play-26/src/main/scala/uk/gov/hmrc/play/bootstrap/package.scala#L89)